src/bridging/providers/across/AcrossBridgeProvider.ts

```diff
@@ -71,19 +74,26 @@ export class AcrossBridgeProvider implements BridgeProvider<AcrossQuoteResult> {
   }
 
   async getBuyTokens(param: GetBuyTokensParams): Promise<TokenInfo[]> {
+    if (!this.options.getTokenInfos) {
+      throw new Error("'getTokenInfos' parameter is required for AcrossBridgeProvider constructor")
+    }
```

Comment:
I don't get it. If it is required, why make the type optional? 
Also, shouldn't this logic be in the constructor instead of this method?

Reply:
Good question :) 

Cause buy token is a method from the SDK, but it is not the most important. 
I tried to relax the options as much as possible. So if you just want to bridge (main feature of the SDK), we don't need the `getTokensInfo`

Anyways, I think this part could use... another new SDK and we can relax even more. But this is completely out of scope